server/src/test/java/org/elasticsearch/common/lucene/uid/VersionsTests.java

```diff
@@ -197,15 +197,15 @@ public void testLuceneVersionOnUnknownVersions() {
                 version.luceneVersion);
```

Comment:
Hi @jpountz, Do you feel that the other tests here cover what this test against 6.x is doing?

Reply:
+1

Reply:
thanks Adrien!